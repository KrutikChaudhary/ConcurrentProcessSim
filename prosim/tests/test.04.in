1 5
Proc1 12 1
DOOP 4
LOOP 5
  BLOCK 3
  DOOP 4
  LOOP 5
    BLOCK 6
    DOOP 7
  END
END
BLOCK 2
DOOP 3
HALT