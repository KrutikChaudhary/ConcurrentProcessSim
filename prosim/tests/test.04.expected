00000: process 1 new
00000: process 1 ready
00000: process 1 running
00004: process 1 blocked
00007: process 1 ready
00007: process 1 running
00011: process 1 blocked
00017: process 1 ready
00017: process 1 running
00022: process 1 ready
00022: process 1 running
00024: process 1 blocked
00030: process 1 ready
00030: process 1 running
00035: process 1 ready
00035: process 1 running
00037: process 1 blocked
00043: process 1 ready
00043: process 1 running
00048: process 1 ready
00048: process 1 running
00050: process 1 blocked
00056: process 1 ready
00056: process 1 running
00061: process 1 ready
00061: process 1 running
00063: process 1 blocked
00069: process 1 ready
00069: process 1 running
00074: process 1 ready
00074: process 1 running
00076: process 1 blocked
00079: process 1 ready
00079: process 1 running
00083: process 1 blocked
00089: process 1 ready
00089: process 1 running
00094: process 1 ready
00094: process 1 running
00096: process 1 blocked
00102: process 1 ready
00102: process 1 running
00107: process 1 ready
00107: process 1 running
00109: process 1 blocked
00115: process 1 ready
00115: process 1 running
00120: process 1 ready
00120: process 1 running
00122: process 1 blocked
00128: process 1 ready
00128: process 1 running
00133: process 1 ready
00133: process 1 running
00135: process 1 blocked
00141: process 1 ready
00141: process 1 running
00146: process 1 ready
00146: process 1 running
00148: process 1 blocked
00151: process 1 ready
00151: process 1 running
00155: process 1 blocked
00161: process 1 ready
00161: process 1 running
00166: process 1 ready
00166: process 1 running
00168: process 1 blocked
00174: process 1 ready
00174: process 1 running
00179: process 1 ready
00179: process 1 running
00181: process 1 blocked
00187: process 1 ready
00187: process 1 running
00192: process 1 ready
00192: process 1 running
00194: process 1 blocked
00200: process 1 ready
00200: process 1 running
00205: process 1 ready
00205: process 1 running
00207: process 1 blocked
00213: process 1 ready
00213: process 1 running
00218: process 1 ready
00218: process 1 running
00220: process 1 blocked
00223: process 1 ready
00223: process 1 running
00227: process 1 blocked
00233: process 1 ready
00233: process 1 running
00238: process 1 ready
00238: process 1 running
00240: process 1 blocked
00246: process 1 ready
00246: process 1 running
00251: process 1 ready
00251: process 1 running
00253: process 1 blocked
00259: process 1 ready
00259: process 1 running
00264: process 1 ready
00264: process 1 running
00266: process 1 blocked
00272: process 1 ready
00272: process 1 running
00277: process 1 ready
00277: process 1 running
00279: process 1 blocked
00285: process 1 ready
00285: process 1 running
00290: process 1 ready
00290: process 1 running
00292: process 1 blocked
00295: process 1 ready
00295: process 1 running
00299: process 1 blocked
00305: process 1 ready
00305: process 1 running
00310: process 1 ready
00310: process 1 running
00312: process 1 blocked
00318: process 1 ready
00318: process 1 running
00323: process 1 ready
00323: process 1 running
00325: process 1 blocked
00331: process 1 ready
00331: process 1 running
00336: process 1 ready
00336: process 1 running
00338: process 1 blocked
00344: process 1 ready
00344: process 1 running
00349: process 1 ready
00349: process 1 running
00351: process 1 blocked
00357: process 1 ready
00357: process 1 running
00362: process 1 ready
00362: process 1 running
00364: process 1 blocked
00366: process 1 ready
00366: process 1 running
00369: process 1 finished
Process 1: Run time 202, Block time 167, Wait time 0
